internal/platform/ipu.go

```diff
@@ -58,8 +58,11 @@ func (pi *IntelDetector) IsDpuPlatform() (bool, error) {
 }
 
 func (pi *IntelDetector) VspPlugin(dpuMode bool, vspImages map[string]string, client client.Client) *plugin.GrpcPlugin {
-	intelVspImage := vspImages["IntelVspImage"]
-	return plugin.NewGrpcPlugin(dpuMode, client, plugin.WithVspImage(intelVspImage))
+	image := vspImages["IntelDetector"]
+	render_data := plugin.CreateCommonData(image)
+	render_data["Command"] = `[ "/usr/bin/ipuplugin" ]`
+	render_data["Args"] = `[ "--bridgeType=linux", "--interface=enp0s1f0d3", "--portMuxVsi=0x0e", "--ovsCliDir=/opt/p4/p4-cp-nws", "-v=debug" ]`
```

Comment:
According to Intel, these args are no longer required....we can leave debug just for debug-ability: https://github.com/intel/ipu-opi-plugins/commit/5d07f00942448bd02dbd54dd0207696a537f64e6#diff-4de381419f5248baa3c90d5e7189bb5770d291fa40f7ba3c0918c31b6f42d61dR32